io/emoncms/88-emoncms.js

```diff
@@ -80,4 +80,52 @@ module.exports = function(RED) {
         });
     }
     RED.nodes.registerType("emoncms",Emoncms);
+ 
+    function Emoncmsin(n) {
+        RED.nodes.createNode(this,n);
+        this.emonServer = n.emonServer;
+        var sc = RED.nodes.getNode(this.emonServer);
+
+        this.baseurl = sc.server;
+        this.apikey = sc.credentials.apikey;
+
+        this.feedid = n.feedid
+        var node = this;
+        var http;
+        if (this.baseurl.substring(0,5) === "https") { http = require("https"); }
+        else { http = require("http"); }
+        this.on("input", function(msg) {
+            this.url = this.baseurl + '/feed/value.json';
+            this.url += '&apikey='+this.apikey;
+            var feedid = this.feedid || msg.feedid;
+            if (feedid !== "") {
+                this.url += '&id=' + feedid;
+            }
+            if (typeof msg.time !== 'undefined') {
+                this.url += '&time=' + msg.time;
+            }
+            node.log("[emoncms] "+this.url);
+            http.get(this.url, function(res) {
+                node.log("Http response: " + res.statusCode);
```

Comment:
We don't generally log 'success' like this - if it's operating normally then it ought to be quiet